packages/web/src/views/base.pug

```diff
@@ -8,6 +8,21 @@ html(lang="se")
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,400;0,700;1,400&display=swap" rel="stylesheet">
+
+    <meta name="title" content="Förtjänar du Robust-sigillet? Testa om din sajt är tillräckligt robust och säker.">
+    <meta name="description" content="Testa om ni tar säkerhet på allvar och uppfyller de mest grundläggande krav man kan ställa på en webbsida idag.">
+
+    <meta property="og:type" content="website">
+    <meta property="og:url" content="https://robust.netnod.se/">
```

Comment:
Make the '/' route a named route instead and use the same method you used to produce the URLs for the sigil's embed code, then we avoid hardcoding :)